react/features/base/conference/functions.js

```diff
@@ -121,3 +127,26 @@ function _reportError(msg, err) {
     // one.
     console.error(msg, err);
 }
+
+/**
+ * Sets the data like avatar URL, email and display name for the local
+ * participant to the conference.
+ *
+ * @param {JitsiConference} conference - The JitsiConference instance.
+ * @param {Object} state - The whole Redux state.
+ * @returns {void}
+ */
+export function setLocalParticipantData(conference, state) {
```

Comment:
Isn't it a bit weird to get the conference object here, when it can be gathered from the state?

Reply:
It is kind of weird, but this function currently is called also from conference.js. I need to verify if the conference will be in the state already at that point and if not how to fix that.

Reply:
Even on react this function is called as part of "createConference" action, so it will not be reduced yet.

Reply:
ack